helm/cluster-vsphere/templates/kubeadmcontrolplane.yaml

```
@@ -13,6 +12,7 @@ spec:
         certSANs:
         - localhost
         - 127.0.0.1
+        - {{ .Values.connectivity.network.controlPlaneEndpoint }}
```

```suggestion
        - {{ .Values.connectivity.network.controlPlaneEndpoint.host }}
```
https://github.com/giantswarm/cluster-vsphere/blob/main/helm/cluster-vsphere/values.yaml#L103